stdlib/3.9/zoneinfo/__init__.pyi

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import os
+import typing
+from datetime import datetime, tzinfo
+from typing import IO, Optional, Sequence, Type, Union
+
+class ZoneInfo(tzinfo):
+    _T = typing.TypeVar("_T", bound="ZoneInfo")
+    key: str
+    def __init__(self, key: str) -> None: ...
+    @classmethod
+    def no_cache(cls: Type[_T], key: str) -> _T: ...
+    @classmethod
+    def from_file(
+        cls: Type[_T], fobj: IO[bytes], /, key: Optional[str] = None
+    ) -> _T: ...
+    @classmethod
+    def clear_cache(cls, *, only_keys=Sequence[str]) -> None: ...
+
+# Note: Both here and in clear_cache, the types allow the use of `str` where
+# a sequence of strings is required. This should be remedied if a solution
+# to this typing bug is found: https://github.com/python/typing/issues/256
+def reset_tzpath(
+    to: Optional[Sequence[Union[os.PathLike, str]]] = None
```

Comment:
```suggestion
    to: Optional[Iterable[Union[os.PathLike, str]]] = ...
```

Reply:
The PEP specifies that this is a `Sequence` rather than an `Iterable`: https://www.python.org/dev/peps/pep-0615/#reset-tzpath-function

(`clear_cache` does specify an `Iterable`).

I don't think it's critical but I'd rather start strict and lessen the constraints later if necessary rather than the other way around. (I think we maybe should make `reset_tzpath` reject anything without a length before the next beta release, so that this is also enforced at runtime).

Reply:
Oh, one question: why the `= ...` instead of `= None`?

Reply:
Your reasoning makes sense.

Defaults don't matter for type checking. For consistency's sake, flake8-pyi enforces a default of `...`. It's one less thing to check and maintain in stubs.